### ISSUE TITLE
SSLError with the example code

### ISSUE THREAD

**Issue by lee-hodg** [mixpanel/mixpanel-python#97]
Following the code in the README 

```
from mixpanel import Mixpanel
mp = Mixpanel(MIXPANEL_TOKEN)
mp.track(USER_ID, 'Upload Done')
```

I get:

```
MixpanelException: HTTPSConnectionPool(host='api-eu.mixpanel.com', port=443): Max retries exceeded with url: /track (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",),))
```

I'm using Mixpanel 4.8.2, python '3.6.10 (default, Jun  6 2020, 21:56:05) \n[GCC 9.3.0]', certifi = ">=2017.4.17", requests 2.20.1, urllib3==1.24.1, Ubuntu 20.04.2 LTS. I also tried upgrading certifi to "2020.12.5"  but that did not help.

If I try with

```
mp = Mixpanel(MIXPANEL_TOKEN, consumer=Consumer(verify_cert=False))
```

then it works (albeit with a warning).

**Comment by seizethedave** [comment]
Yeah, it's an unfortunate known snag. It is enough of a friction point that I plan to turn off cert verification by default.

**Comment by asymness** [comment]
+1, this is blocking me from using Mixpanel.

**Comment by seizethedave** [comment]
@asymness try the `verify_cert` initializer arg for now. I’ll put out a new version this week.

**Comment by seizethedave** [comment]
Fixed in 4.8.3: https://github.com/mixpanel/mixpanel-python/releases/tag/4.8.3
